debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/util/BoundedLinkedHashSet.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I think instead of a new class we can re-use `org.apache.kafka.common.cache.LRUCache<K, V>` as we do in other parts of the codebase.  The `LinkedHashSet` uses `LinkedHashMap` interanally anyway so there will not be any performance loss.